e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,131 @@
+        getAttachmentPost().eq(0).invoke('height').as('initialAttachmentHeight');
+        getInlinImgPost().eq(0).invoke('height').as('initialInlineImgHeight');
+        getFirstTextPost().eq(0).invoke('height').as('initial1stPostHeight');
+        getLastTextPost().eq(0).invoke('height').as('initiallastPostHeight');
+        getGifPost().eq(0).invoke('height').as('initialGifPostHeight');
+        getJpgPost().eq(0).invoke('height').as('initialJpgPostHeight');
+
+        // # Switch the account settings for the test user to enable Fixed width center
+        cy.toAccountSettingsModal();
+        cy.get('#accountSettingsModal').should('be.visible').within(() => {
+            cy.findByText('Display', {timeout: timeouts.ONE_MIN}).click();
+            cy.findByText('Channel Display').click();
+            cy.findByLabelText('Fixed width, centered').click();
+            cy.findByText('Save').click();
+            cy.get('#accountSettingsHeader > .close').click();
+        });
+
+        // # Browse to Channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+        // * Verify there is no scroll pop
+        cy.get('#post-list').should('exist').within(() => {
+            cy.get('@initialUserNameHeight').then((originalHeight) => {
+                getUserNameTitle().eq(0).should('exist').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initial1stPostHeight').then((originalHeight) => {
+                getFirstTextPost().eq(0).should('exist').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initiallastPostHeight').then((originalHeight) => {
+                getLastTextPost().eq(0).should('exist').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialThumbnailHeight').then((originalHeight) => {
+                getThumbnailPost().should('have.length', '2').and('have.css', 'height', originalHeight + 'px');
+            });
+            cy.get('@initialInlineImgHeight').then((originalHeight) => {
+                getInlinImgPost().should('have.css', 'height', (originalHeight + 2) + 'px');
+            });
+            cy.get('@initialAttachmentHeight').then((originalHeight) => {
+                getAttachmentPost().should('have.css', 'height', (originalHeight + 2) + 'px');
+            });
+
+            cy.get('@initialGifPostHeight').then((originalHeight) => {
+                getGifPost().should('have.css', 'height', originalHeight + 'px');
+            });
+
+            cy.get('@initialJpgPostHeight').then((originalHeight) => {
+                getJpgPost().should('have.css', 'height', originalHeight + 'px');
+            });
+        });
+
+        // * Verify All posts are displayed correctly
+        cy.findAllByTestId('postContent').should('have.length', '9').and('have.class', 'post__content center');
+    });
+
+    const getUserNameTitle = () => {
+        return cy.findAllByLabelText('sysadmin');
```

Comment:
Why not `cy.findByLabelText`?

Reply:
cy.findByLabelText throwing error saying found multiple elements

![image](https://user-images.githubusercontent.com/6909890/93692602-1613c480-faaa-11ea-9fd5-dfb7508dbf2f.png)